Junchao-Mellanox/sonic-mgmt
# [snmp] Add test cases for SONiC physical entity MIB feature

### Description of PR


Summary:
Add test cases for SONiC physical entity MIB feature

### Type of change




### Approach
#### What is the motivation for this PR?

There is no regression test case for physical entity MIB before, and we create a few new test cases to cover the existing/new MIB objects.

#### How did you do it?

1. Adjust snmp_facts.py to support new MIB objects
2. Add test case to verify transceiver related entities
3. Add test case to verify PSU related entities
4. Add test case to verify fan related entities
5. Add test case to verify thermal related entities
6. Add test cases to verify all new platform API introduced in this feature

#### How did you verify/test it?

Run the new regression test cases 

#### Any platform specific information?

N/A

#### Supported testbed topology if it's a new test case?

N/A

### Documentation
